### ISSUE TITLE
Nothing happens after confirmation on Facebook

### ISSUE THREAD

**Issue by blastmasta** [roughike/flutter_facebook_login#193]
1. The first screen showing choices **"Log in with the Facebook app"** or **"Log in with phone or email"**. I click on **"Log in with the Facebook app"**:

2. On the second screen, I click on **"Continue"**.

3. And it returns me on the first screen, and it not closing, nothing happens after.

**Comment by TsogoS** [comment]
Yeah same here...  Using 1.2.0 and implementation 'com.facebook.android:facebook-android-sdk:[5,6)'
